.github/workflows/ci-python-sdk.yaml

```diff
@@ -601,6 +602,7 @@ jobs:
           name: constraints-${{ matrix.python }}-${{ matrix.airflow }}
           path: ./python-sdk/constraints-${{ matrix.python }}-${{ matrix.airflow }}
           if-no-files-found: error
+          include-hidden-files: true
 
   Code-Coverage:
     if: github.event.action != 'labeled'
```

Comment:
I just noticed some tests failed due to uploading artefacts with the same name; this has to be addressed before we merge this PR. This project thas not been actively maintained for a while, so we may uncover other issues before this PR can be merged.